spec/models/product_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe Product, type: :model do
+
+  empty_product = Product.new
+
+  let(:good_product) do
```

Comment:
I'd suggest using `valid_product` and `invalid_product` rather than `good_product` and `bad_product`. Descriptive naming is *super* helpful when you're working in a larger code base... or looking at code you wrote a while ago. :wink: